docs/src/content/pages/collections-and-singletons.mdoc

```diff
@@ -1,5 +1,8 @@
 ---
 title: Collections & singletons
+summary: >-
+  Keystatic has two concepts or mechanisms to manage data: collections and
+  singletons.
```

Comment:
A bunch of these summaries have an extra line break for the last word or last few words. Is that intentional?

Reply:
Nah let me go through and see if I can fix that 🙏🏻

Reply:
Actually, I added all these through the Admin UI so it seems if the string is a certain length then it will populate the mdoc like this. Still think we need to tweak these @BarnabyBishop?

Reply:
Probably not worth it if the AdminUI will just wrap them again when we make changes. Good to leave for now